makefile

```diff
@@ -35,6 +43,36 @@ release: | tmp/output/release/
 	@test $(VERSION)
 	gh release create $(VERSION) --notes "New release." --title "$(VERSION)" tmp/output/release/*
 
+# Mappings
+# todo: I'd like to do USE_PREBUILT=false, but this error is occuring:
+#  cp: cannot stat 'tmp/input/mondo/src/ontology/mappings/mondo.sssom.tsv': No such file or directory
+#  ...even though I checked and the file at that path does exist.
+USE_PREBUILT=true
+
+tmp/input/mondo/:
+	rm -rf $@ &&\
+	cd tmp/input/ &&\
+	git clone --depth 1 https://github.com/monarch-initiative/mondo
+
+tmp/input/mondo.sssom.tsv: tmp/input/mondo/
+	if [ $(USE_PREBUILT) = true ]; then wget https://raw.githubusercontent.com/monarch-initiative/mondo/master/src/ontology/mappings/mondo.sssom.tsv -O $@; else cd tmp/input/ &&\
+		cd mondo/src/ontology &&\
+		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
+		cp tmp/input/mondo/src/ontology/mappings/mondo.sssom.tsv $@; fi
+
+# todo: Stable URI/filename issue: https://github.com/monarch-initiative/icd11/pull/12#discussion_r1542187711
+tmp/input/Orphanet_Nomenclature_Pack_EN/ORPHA_ICD11_mapping_en_newversion_2023.xml: | tmp/input/
+	wget https://www.orphadata.com/data/nomenclature/packs/Orphanet_Nomenclature_Pack_EN.zip -O tmp/input/Orphanet_Nomenclature_Pack_EN.zip
+	unzip tmp/input/Orphanet_Nomenclature_Pack_EN.zip -d tmp/input/Orphanet_Nomenclature_Pack_EN
+
+tmp/output/release/mondo-icd11.sssom.tsv tmp/output/release/ordo-icd11.sssom.tsv: tmp/input/Orphanet_Nomenclature_Pack_EN/ORPHA_ICD11_mapping_en_newversion_2023.xml tmp/input/mondo.sssom.tsv | tmp/output/release/
```

Comment:
## `mondo-icd11foundation.sssom.tsv`; where to put it?
- [ ] @matentzn and @twhetzel 

I'm going to add it to the release. But what else?

I am thinking that maybe it should go in the `mondo` repo, in `src/ontology/mappings/`?

```
cd src/ontology/mappings; ls
mondo.sssom.tsv                       mondo_exactmatch_icd10who.sssom.tsv   mondo_hasdbxref_icd10cm.sssom.tsv     mondo_hasdbxref_obi.sssom.tsv         mondo_narrowmatch_orphanet.sssom.tsv
```

- [ ] If, so, I think I need to rename it to `mondo_exactmatch_icd11foundation.sssom.tsv`, yes?

Also, if I do this, then this would be the only SSSOM file in the `mondo` repo that is copied over into this directory from an external repository. All of the other files there are generated by some process I'm currently a little in the dark on.

Reply:
I'll need to leave this to Nico since I don't know how this is handled yet.

Reply:
Sabrina did ask us for a robot template. For now, she has created it on her own. There is still the question as to whether or not I should automate that.

Reply:
Nothing is done with the SSSOM file itself. It's just an intermediary format for the robot template.

- [ ] Consider: general tool for converting SSSOM to robot template
  - Perhaps in `mondolib`. Could be a `make` command.
  - Sometimes there will be additional fields that need to get added to the robot template. Would not be part of this conversion. Would be added post facto.